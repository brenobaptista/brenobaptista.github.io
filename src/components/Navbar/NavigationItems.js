import React from 'react'
import NavigationItem from './NavigationItem'

const NavigationItems = () => (
  <ul className='navigationItems'>
    <NavigationItem link='/'>
      HOME
    </NavigationItem>
    <NavigationItem link='/projects'>
      PROJECTS
    </NavigationItem>
    <NavigationItem link='/blog'>
      BLOG
    </NavigationItem>
  </ul>
)

export default NavigationItems
