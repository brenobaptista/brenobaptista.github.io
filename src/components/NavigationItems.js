import React from 'react';
import NavigationItem from './NavigationItem';

const navigationItems = () => (
  <ul className="navigationItems">
    <NavigationItem link="/">HOME</NavigationItem>
    <NavigationItem link="/projects">PROJECTS</NavigationItem>
    <NavigationItem link="/stack">STACK</NavigationItem>
    <NavigationItem link="/downloads">DOWNLOADS</NavigationItem>
  </ul>
);

export default navigationItems;
