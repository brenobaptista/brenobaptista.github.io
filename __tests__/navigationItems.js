import React from 'react'
import { render } from '@testing-library/react'
import NavigationItems from '../src/components/Navbar/NavigationItems'

test('matches snapshot', () => {
  const clicked = jest.fn()
  const { asFragment } = render(<NavigationItems clicked={clicked} email='' />)

  expect(asFragment(<NavigationItems clicked={clicked} email='' />)).toMatchSnapshot()
})
