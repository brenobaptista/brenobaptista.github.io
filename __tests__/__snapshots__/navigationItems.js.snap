// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <ul
    class="navigationItems"
  >
    <li
      class="navigationItem"
    >
      <a
        href="/"
      >
        HOME
      </a>
    </li>
    <li
      class="navigationItem"
    >
      <a
        href="/projects"
      >
        PROJECTS
      </a>
    </li>
    <li
      class="navigationItem"
    >
      <a
        href="/blog"
      >
        BLOG
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
